src/core_atmosphere/physics/physics_wrf/module_mynnsfclay_driver.F90

```diff
@@ -194,7 +194,8 @@ subroutine mynnsfclay_driver(                                   &
  integer,intent(in):: ids,ide,jds,jde,kds,kde, &
                       ims,ime,jms,jme,kms,kme, &
                       its,ite,jts,jte,kts,kte
- integer,intent(in):: itimestep
+ logical,intent(in):: restart,cycling
+ integer,intent(in):: itimestep, initflag
```

Comment:
normally I would say to add initflag to its own line to preserve the original code, but this file doesn't exist in the ncar repo so it probably doesn't matter

Reply:
done anyway